docs/ru/extended_roadmap.md

```diff
@@ -9,22 +9,24 @@
 
 ### 1.1. Индексы по z-Order curve, normalized z-Order curve.
 
+[#6286](https://github.com/ClickHouse/ClickHouse/pull/6286)
+
 Задача также относится к категории "17. Работа с географическими данными", так как geoHash - это частный случай z-Order curve.
 Также связана с задачей 24.27 для нечёткого поиска полудубликатов строк, так как позволит индексировать min-hash.
 Задача "normalized z-Order curve" в перспективе может быть полезна для БК и Метрики, так как позволяет смешивать OrderID и PageID и избежать дублирования данных.
 В задаче также вводится способ индексации путём обращения функции нескольких аргументов на интервале, что имеет смысл для дальнейшего развития.
 
 Изначально делал [Андрей Чулков](https://github.com/achulkov2), ВШЭ, теперь доделывает [Ольга Хвостикова](https://github.com/stavrolia), но сроки немного сдвинуты из-за задачи 25.9. Будем надеятся на реализацию к концу ноября. Впрочем, [Андрей Чулков](https://github.com/achulkov2) скоро сможет помочь её доделать.
 
-Upd. Доделывать будет другой человек. Хотелось бы в Q1, но приоритет не высокий.
+Upd. Доделывать будет другой человек. Приоритет не высокий.
 
 ### 1.2. Wait-free каталог баз данных.
 
 Q1. Делает [Александр Токмаков](https://github.com/tavplubix), первый рабочий вариант в декабре 2019. Нужно для DataLens и Яндекс.Метрики.
 
 Манипуляции с каталогом баз данных: запросы CREATE TABLE, DROP TABLE, RENAME TABLE и DATABASE, требуют синхронизации с помощью блокировок. Эта синхронизация становится весьма сложной, так как на неё полагается много внутренних структур данных.
 
-Предлагается реализовать альтернативный подход, в котором таблицы и базы данных являются всего лишь ссылками на persistent объекты. Подробное описание задачи: https://github.com/yandex/ClickHouse/issues/6787
+Предлагается реализовать альтернативный подход, в котором таблицы и базы данных являются всего лишь ссылками на persistent объекты. Подробное описание задачи: [#6787](https://github.com/yandex/ClickHouse/issues/6787)
```

Comment:
Our website engine should automatically detect links. Manual correction of all links does not make sense.